### ISSUE TITLE
Clang crash:  Assertion `It != MinBWs.end() && "Expected item in MinBWs."' failed.

### ISSUE THREAD

**Issue by shao-hua-li** [llvm/llvm-project#72833]
Clang at -O3 crashes on the following test case.

Bisected to 5adfad254edb765fbbabdf5e7521cfb3111dfeb1, which was committed by @alexey-bataev 

Compiler explorer: https://godbolt.org/z/hqGzds3e3

```console
% cat a.c
int a, b, c;
long d, e;
short g;
char h;
static long *i = &d;
short(j)(short k, short l) { return k * l; }
static void m(long k, short l) {
  unsigned short n;
  for (;;) {
    e = 0;
    for (; e <= 2; e++) {
      h = 0;
      for (; h <= 2; h++) {
        c = 2;
        for (; c >= 0; c--) {
          int *o = &b;
          int f = l;
          while (f) {
            a++;
            f >>= 1;
          }
          n = j(*i & a + 3LL, k);
          *o |= n;
        }
      }
      for (; d <= 2; d++)
        ;
    }
    break;
  }
}
void p() {
  unsigned q = (unsigned char)-1;
  m(q, (char)g + 176);
}
int main() {}
%
% clang -O3 a.c 
clang-18: /tmp/tmpk3k1m280/tmpjnxx63qh/llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp:11392: llvm::Value* llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool): Assertion `It != MinBWs.end() && "Expected item in MinBWs."' failed.
PLEASE submit a bug report to https://github.com/llvm/llvm-project/issues/ and include the crash backtrace, preprocessed source, and associated run script.
Stack dump:
0.      Program arguments: /zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/clang-18 -cc1 -triple x86_64-unknown-linux-gnu -emit-obj -dumpdir a- -disable-free -clear-ast-before-backend -main-file-name a.c -mrelocation-model pic -pic-level 2 -pic-is-pie -mframe-pointer=none -fmath-errno -ffp-contract=on -fno-rounding-math -mconstructor-aliases -funwind-tables=2 -target-cpu x86-64 -tune-cpu generic -debugger-tuning=gdb -fdebug-compilation-dir=/zdata/shaoli/realsmith/watchdir/reduced/case_aDhi8Ptd -fcoverage-compilation-dir=/zdata/shaoli/realsmith/watchdir/reduced/case_aDhi8Ptd -resource-dir /zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/lib/clang/18 -internal-isystem /zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/lib/clang/18/include -internal-isystem /usr/local/include -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/11/../../../../x86_64-linux-gnu/include -internal-externc-isystem /usr/include/x86_64-linux-gnu -internal-externc-isystem /include -internal-externc-isystem /usr/include -O3 -ferror-limit 19 -fgnuc-version=4.2.1 -fcolor-diagnostics -vectorize-loops -vectorize-slp -faddrsig -D__GCC_HAVE_DWARF2_CFI_ASM=1 -o /tmp/a-82e8f4.o -x c a.c
1.      <eof> parser at end of file
2.      Optimizer
 #0 0x00007efc3c82347f llvm::sys::PrintStackTrace(llvm::raw_ostream&, int) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x93147f)
 #1 0x00007efc3c8209a4 SignalHandler(int) Signals.cpp:0:0
 #2 0x00007efc3bb07090 (/lib/x86_64-linux-gnu/libc.so.6+0x43090)
 #3 0x00007efc3bb0700b raise /build/glibc-BHL3KM/glibc-2.31/signal/../sysdeps/unix/sysv/linux/raise.c:51:1
 #4 0x00007efc3bae6859 abort /build/glibc-BHL3KM/glibc-2.31/stdlib/abort.c:81:7
 #5 0x00007efc3bae6729 get_sysdep_segment_value /build/glibc-BHL3KM/glibc-2.31/intl/loadmsgcat.c:509:8
 #6 0x00007efc3bae6729 _nl_load_domain /build/glibc-BHL3KM/glibc-2.31/intl/loadmsgcat.c:970:34
 #7 0x00007efc3baf7fd6 (/lib/x86_64-linux-gnu/libc.so.6+0x33fd6)
 #8 0x00007efc3e3a0118 llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae118)
 #9 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#10 0x00007efc3e39c4dc llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa4dc)
#11 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#12 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#13 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#14 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#15 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#16 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#17 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#18 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#19 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#20 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#21 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#22 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#23 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#24 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#25 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#26 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#27 0x00007efc3e3a032f llvm::slpvectorizer::BoUpSLP::vectorizeOperand(llvm::slpvectorizer::BoUpSLP::TreeEntry*, unsigned int, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ae32f)
#28 0x00007efc3e39c45f llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::slpvectorizer::BoUpSLP::TreeEntry*, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24aa45f)
#29 0x00007efc3e3a30d2 llvm::slpvectorizer::BoUpSLP::vectorizeTree(llvm::MapVector<llvm::Value*, llvm::SmallVector<llvm::Instruction*, 2u>, llvm::DenseMap<llvm::Value*, unsigned int, llvm::DenseMapInfo<llvm::Value*, void>, llvm::detail::DenseMapPair<llvm::Value*, unsigned int>>, llvm::SmallVector<std::pair<llvm::Value*, llvm::SmallVector<llvm::Instruction*, 2u>>, 0u>> const&, llvm::SmallVectorImpl<std::pair<llvm::Value*, llvm::Value*>>&, llvm::Instruction*) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24b10d2)
#30 0x00007efc3e3a5b7c llvm::slpvectorizer::BoUpSLP::vectorizeTree() (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24b3b7c)
#31 0x00007efc3e3b2452 llvm::SLPVectorizerPass::tryToVectorizeList(llvm::ArrayRef<llvm::Value*>, llvm::slpvectorizer::BoUpSLP&, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24c0452)
#32 0x00007efc3e3b4679 llvm::SLPVectorizerPass::tryToVectorize(llvm::Instruction*, llvm::slpvectorizer::BoUpSLP&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24c2679)
#33 0x00007efc3e3b4863 llvm::SLPVectorizerPass::tryToVectorize(llvm::ArrayRef<llvm::WeakTrackingVH>, llvm::slpvectorizer::BoUpSLP&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24c2863)
#34 0x00007efc3e3b50ec llvm::SLPVectorizerPass::vectorizeRootInstruction(llvm::PHINode*, llvm::Instruction*, llvm::BasicBlock*, llvm::slpvectorizer::BoUpSLP&, llvm::TargetTransformInfo*) (.constprop.0) SLPVectorizer.cpp:0:0
#35 0x00007efc3e3b8337 llvm::SLPVectorizerPass::vectorizeChainsInBlock(llvm::BasicBlock*, llvm::slpvectorizer::BoUpSLP&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24c6337)
#36 0x00007efc3e3bb906 llvm::SLPVectorizerPass::runImpl(llvm::Function&, llvm::ScalarEvolution*, llvm::TargetTransformInfo*, llvm::TargetLibraryInfo*, llvm::AAResults*, llvm::LoopInfo*, llvm::DominatorTree*, llvm::AssumptionCache*, llvm::DemandedBits*, llvm::OptimizationRemarkEmitter*) (.part.0) SLPVectorizer.cpp:0:0
#37 0x00007efc3e3bc283 llvm::SLPVectorizerPass::run(llvm::Function&, llvm::AnalysisManager<llvm::Function>&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x24ca283)
#38 0x00007efc3f73cc06 llvm::detail::PassModel<llvm::Function, llvm::SLPVectorizerPass, llvm::PreservedAnalyses, llvm::AnalysisManager<llvm::Function>>::run(llvm::Function&, llvm::AnalysisManager<llvm::Function>&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0x384ac06)
#39 0x00007efc43270d15 llvm::detail::PassModel<llvm::Function, llvm::PassManager<llvm::Function, llvm::AnalysisManager<llvm::Function>>, llvm::PreservedAnalyses, llvm::AnalysisManager<llvm::Function>>::run(llvm::Function&, llvm::AnalysisManager<llvm::Function>&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x2152d15)
#40 0x00007efc3ca3d98d llvm::ModuleToFunctionPassAdaptor::run(llvm::Module&, llvm::AnalysisManager<llvm::Module>&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0xb4b98d)
#41 0x00007efc4325e346 llvm::detail::PassModel<llvm::Module, llvm::ModuleToFunctionPassAdaptor, llvm::PreservedAnalyses, llvm::AnalysisManager<llvm::Module>>::run(llvm::Module&, llvm::AnalysisManager<llvm::Module>&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x2140346)
#42 0x00007efc3ca3b8bd llvm::PassManager<llvm::Module, llvm::AnalysisManager<llvm::Module>>::run(llvm::Module&, llvm::AnalysisManager<llvm::Module>&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libLLVM-18git.so+0xb498bd)
#43 0x00007efc432724bf (anonymous namespace)::EmitAssemblyHelper::RunOptimizationPipeline(clang::BackendAction, std::unique_ptr<llvm::raw_pwrite_stream, std::default_delete<llvm::raw_pwrite_stream>>&, std::unique_ptr<llvm::ToolOutputFile, std::default_delete<llvm::ToolOutputFile>>&, clang::BackendConsumer*) BackendUtil.cpp:0:0
#44 0x00007efc43275628 (anonymous namespace)::EmitAssemblyHelper::EmitAssembly(clang::BackendAction, std::unique_ptr<llvm::raw_pwrite_stream, std::default_delete<llvm::raw_pwrite_stream>>, clang::BackendConsumer*) BackendUtil.cpp:0:0
#45 0x00007efc43275c9b clang::EmitBackendOutput(clang::DiagnosticsEngine&, clang::HeaderSearchOptions const&, clang::CodeGenOptions const&, clang::TargetOptions const&, clang::LangOptions const&, llvm::StringRef, llvm::Module*, clang::BackendAction, llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem>, std::unique_ptr<llvm::raw_pwrite_stream, std::default_delete<llvm::raw_pwrite_stream>>, clang::BackendConsumer*) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x2157c9b)
#46 0x00007efc43734186 clang::BackendConsumer::HandleTranslationUnit(clang::ASTContext&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x2616186)
#47 0x00007efc41c48689 clang::ParseAST(clang::Sema&, bool, bool) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0xb2a689)
#48 0x00007efc43733698 clang::CodeGenAction::ExecuteAction() (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x2615698)
#49 0x00007efc443d2019 clang::FrontendAction::Execute() (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x32b4019)
#50 0x00007efc4434c97e clang::CompilerInstance::ExecuteAction(clang::FrontendAction&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x322e97e)
#51 0x00007efc4446ae66 clang::ExecuteCompilerInvocation(clang::CompilerInstance*) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/../lib/libclang-cpp.so.18git+0x334ce66)
#52 0x0000559bb6b5da46 cc1_main(llvm::ArrayRef<char const*>, char const*, void*) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/clang-18+0x15a46)
#53 0x0000559bb6b5538a ExecuteCC1Tool(llvm::SmallVectorImpl<char const*>&, llvm::ToolContext const&) driver.cpp:0:0
#54 0x0000559bb6b59fcc clang_main(int, char**, llvm::ToolContext const&) (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/clang-18+0x11fcc)
#55 0x0000559bb6b54e05 main (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/clang-18+0xce05)
#56 0x00007efc3bae8083 __libc_start_main /build/glibc-BHL3KM/glibc-2.31/csu/../csu/libc-start.c:342:3
#57 0x0000559bb6b54e4e _start (/zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin/clang-18+0xce4e)
clang: error: unable to execute command: Aborted
clang: error: clang frontend command failed due to signal (use -v to see invocation)
clang version 18.0.0 (https://github.com/llvm/llvm-project.git 5adfad254edb765fbbabdf5e7521cfb3111dfeb1)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /zdata/shaoli/compilers/ccbuilder-compilers/clang-5adfad254edb765fbbabdf5e7521cfb3111dfeb1/bin
clang: note: diagnostic msg:
********************

PLEASE ATTACH THE FOLLOWING FILES TO THE BUG REPORT:
Preprocessed source(s) and associated run script(s) are located at:
clang: note: diagnostic msg: /tmp/a-617235.c
clang: note: diagnostic msg: /tmp/a-617235.sh
clang: note: diagnostic msg:

********************
%
```

**Comment by dtcxzyw** [comment]
Reduced test case: https://godbolt.org/z/jGoP8PfME

**Comment by alexey-bataev** [comment]
Fixed in f609d4ba1d940c781f4fed44f7c69422d1766f09
